src/dev/flang/ast/Types.java

```diff
@@ -283,6 +283,82 @@ public Resolved(SrcModule mod, CreateType ct, AbstractFeature universe)
           res.resolveTypes(t.feature());
         }
     }
+    public static interface LookupFeature
+    {
+      AbstractFeature lookupFeature(AbstractFeature outer, FeatureName fn);
+    }
+    public Resolved(LookupFeature luf, AbstractFeature _universe)
```

Comment:
Can `Resolved(SrcModule mod, CreateType ct, AbstractFeature universe)` be implemented using this new method to avoid code duplication?

Reply:
@fridis I removed the code duplication in a recent commit.